Sources/Stripe/Stripe+Extensions.swift

```diff
@@ -38,3 +38,34 @@ extension Request {
         }
     }
 }
+
+extension StripeClient {
+    public static func verifySignature(for req: Request, secret: String, tolerance: Double = 300) throws {
```

Comment:
Can you just add some documentation here for clarity :)

Reply:
sure thing ;)